JS/02/table/table.js

```diff
@@ -0,0 +1,215 @@
+const users = []
+
+const infoUser = () => {
+  const boxPopup = document.querySelector('.box-popup')
+  const fullName = boxPopup.querySelector('.full-name').value
+  const gender = boxPopup.querySelector('.gender').value
+  const age = boxPopup.querySelector('.age').value
+
+  const user = {
+    fullName: fullName,
+    gender: gender,
+    age: age
+  }
+  users.push(user)
+  render(users)
+}
+
+const render = users => {
+  const thead = document.querySelector('.heading-table')
+  const th = thead.querySelector('th:first-child')
+  th.innerHTML = `<input type="checkbox" name="" id="">`
+
+  const listUser = document.querySelector('.list-user')
+  let html = ''
+  users.forEach((user, i) => {
+    html += `
+      <tr>
+        <td>
+          <input type="checkbox" value="${i}" name="" id="">
+        </td>
+        <td>${i}</td>
+        <td>${user.fullName}</td>
+        <td>${user.gender}</td>
+        <td>${user.age}</td>
+        <td>
+          <div class="box_active-table">
+            <button type="button"
+              onclick="editUser(event, ${i})"
+              class="btn btn-primary box_active-table--edit">
+                Edit
+            </button>
+            <button type="button"
+              onclick="editSave(event, ${i})"
+              class="btn btn-success box_active-table--save">
+                Save
+            </button>
+            <button type="button"
+              onclick="editCancel(event)"
+              class="btn btn-warning box_active-table--cancel">
+                Cancel
+            </button>
+            <button type="button"
+              onclick="deleteUser(${i})"
+              class="btn btn-danger box_active-table--delete">
+                Delete
+            </button>
+          </div>
+        </td>
+      </tr>
+    `
+  })
+  listUser.innerHTML = html
+  listEvent()
```

Comment:
function name should be more meaningful, start with verb
```suggestion
  addListEvents()
  addActionEvents()
```